frontend/containers/forms/filters/component.tsx

```diff
@@ -110,80 +116,148 @@ export const Filters: FC<FiltersProps> = ({
   };
   */
 
-  return isLoading ? (
-    <Loading />
-  ) : (
-    <form
-      className="relative p-6 max-h-[75vh] flex flex-col"
-      onSubmit={handleSubmit(onSubmitFilters)}
+  return (
+    <div
```

Comment:
On smaller screens (e.g. 1280x800), the user is not able to see the whole filters menu:
![The filters menu is cut at the bottom of the screen](https://user-images.githubusercontent.com/6073968/190177767-000e6e39-c12a-488c-8051-621db6b6925b.png)

I recently addressed this in [this commit](https://github.com/Vizzuality/heco-invest/commit/cc17463554052188ddacb34fad20bae4ee30991b) but it seems the change has been lost here.